core/src/State/FocusedElement.ts

```diff
@@ -375,12 +375,26 @@ export class FocusedElementState
                 }
             }
 
-            let fromElement = (isTab && ctx.mover && ctx.moverOptions?.navigationType === Types.MoverKeys.Arrows)
-                ? (isPrev // Find next focusable outside of the Mover container.
-                    ? this._ah.focusable.findFirst(ctx.mover)
-                    : this._ah.focusable.findLast(ctx.mover)
-                )
-                : curElement;
+            let fromElement: HTMLElement | null = curElement;
+
+            // If the current element is in a mover, move to the mover boundaries since a mover is considered a single tabstop
+            if (isTab && ctx.mover && ctx.moverOptions?.navigationType === Types.MoverKeys.Arrows) {
+                // Consider nested movers a as a single tab stop, go up until there is no more mover
+                if (isPrev) {
+                    let  parentCtx: typeof ctx | undefined = ctx;
+                    let rootMover = ctx.mover;
+                    while(parentCtx?.mover && parentCtx?.mover.parentElement) {
+                        rootMover = parentCtx.mover;
+                        parentCtx = RootAPI.getAbilityHelpersContext(this._ah, parentCtx.mover.parentElement);
+                    }
+
+                    fromElement = this._ah.focusable.findFirst(rootMover);
+                } else {
+                    fromElement = this._ah.focusable.findLast(ctx.mover);
```

Comment:
It seems to me that the same root finding logic should be applied for the moving forward :).

Reply:
It's not really necessary, since the way the tree walking works, it will always find the correct boundary element

Reply:
I've added a comment to reflect that

Reply:
Ah i see your point, donw